client/my-sites/people/team-list/team.jsx

```diff
@@ -22,6 +26,40 @@ class Team extends React.Component {
 
 	renderLoadingPeople = () => <PeopleListItem key="people-list-item-placeholder" />;
 
+	getP2headerText = () => {
+		const { p2Guests, totalUsers, isP2HubSite, translate } = this.props;
+		const translateOptions = {
+			args: {
+				numberPeople: totalUsers,
+			},
+			count: totalUsers,
+			context: 'A navigation label.',
+		};
+
+		if ( isP2HubSite ) {
+			return translate(
+				'There is %(numberPeople)d members in this workspace',
+				'There are %(numberPeople)d members in this workspace',
+				translateOptions
+			);
+		}
+
+		if ( p2Guests?.found ) {
+			translateOptions.args.numberGuests = p2Guests?.found;
+			return translate(
+				'There is %(numberPeople)d member and %(numberGuests)d guests in this P2',
+				'There are %(numberPeople)d members and %(numberGuests)d guests in this P2',
```

Comment:
Not sure if this works the way I understand it... but will this cover all the correct pluralization combinations? I notice it's "guests" for both strings, but 'numberGuests' is being passed.

Reply:
You're right. That would be off if there is say 1 member and 3 guests. I pushed a change in fc2944a8942bd74aed26fdac2a538dec9db6f91e that also makes the guests singular/plural. I'm not 100% that the approach will work for all languages. What do you think?

Reply:
Yeah, it's a tough one. I don't see other instances we can crib off of. We can go with it for now and perhaps ask the translations team later.